app/models/plant.rb

```diff
@@ -0,0 +1,3 @@
+class Plant < ApplicationRecord
```

Comment:
This should be a many-to-many relationship, not a one to many. So plant doesn't belong to plot, but instead should have
`has_many :plot_plants`
`has_many :plots, through: :plot_plants`